inference-engine/tests/functional/plugin/shared/include/behavior/core_integration.hpp

```diff
@@ -606,6 +659,24 @@ TEST_P(IEClassNetworkTestP, QueryNetworkWithKSO) {
     }
 }
 
+TEST_P(IEClassMultiDeviceTestQueryNetwork, QueryNetworkActualMultiDeviceNoThrow) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    Core ie = createCoreWithTemplate();
+
+    std::vector<std::string> deviceIDs = ie.GetMetric(deviceNames.front(), METRIC_KEY(AVAILABLE_DEVICES));
+    if (deviceIDs.size() < deviceNames.size())
+        GTEST_SKIP();
+
+    std::string multiDeviceName = CommonTestUtils::DEVICE_MULTI + std::string(":");
+    for (auto& dev_name : deviceNames) {
+        multiDeviceName += dev_name;
+        if (&dev_name != &(deviceNames.back())) {
+            multiDeviceName += ",";
+        }
+    }
+    ASSERT_NO_THROW(ie.QueryNetwork(actualNetwork, multiDeviceName));
+}
```

Comment:
can we move these tests to `ov_core_integration.hpp`? There we test new OV 2.0 API which is future proof